src/frontend/src/pages/TeamsPage/TeamMembersPageBlock.tsx

```diff
@@ -7,118 +7,157 @@ import { Autocomplete, Grid, IconButton, TextField } from '@mui/material';
+      <Grid item xs={8} md={10} lg={11}>
+        <NERAutocomplete
+          id="head-autocomplete"
+          options={headOptions}
+          onChange={(_event, newValue) => newValue && setHead(newValue)}
+          filterSelectedOptions
+          size="small"
+          placeholder="Select a User"
+          value={head}
+          label={'Head'}
+        />
+      </Grid>
+      <Grid container direction={'row'} xs={4} md={2} lg={1}>
+        <Grid item>
+          <IconButton children={<Save />} onClick={submitHead} />
         </Grid>
-        <Grid item xs={12}>
-          <Autocomplete
-            isOptionEqualToValue={(option, value) => option.id === value.id}
-            filterSelectedOptions
-            multiple
-            id="tags-standard"
-            options={options}
-            value={members}
-            onChange={(_event, newValue) => setMembers(newValue)}
-            getOptionLabel={(option) => option.label}
-            renderInput={(params) => (
-              <TextField {...params} variant="standard" label="Members" placeholder="Select A User" />
-            )}
-          />
+        <Grid item>
+          <IconButton children={<Cancel />} onClick={() => setIsEditingHead(false)} />
         </Grid>
       </Grid>
-    </PageBlock>
+    </Grid>
   );
 
-  const nonEditingView = (
-    <PageBlock
-      title={'People'}
-      headerRight={hasPerms ? <IconButton onClick={() => setIsEditingMembers(true)} children={<Edit />} /> : null}
-    >
-      <Grid container spacing={1}>
-        <Grid item xs={12}>
-          <DetailDisplay label="Head" content={fullNamePipe(team.head)} />
-        </Grid>
-        <Grid item xs={12}>
-          <DetailDisplay label="Leads" content={team.leads.map((lead) => fullNamePipe(lead)).join(', ')} />
+  const editingMembersView = (
+    <Grid container direction={'row'} pl={1}>
+      <Grid item xs={8} md={10} lg={11}>
+        <Autocomplete
+          isOptionEqualToValue={(option, value) => option.id === value.id}
+          filterSelectedOptions
+          multiple
+          id="tags-standard"
+          options={memberOptions}
+          value={members}
+          onChange={(_event, newValue) => setMembers(newValue)}
+          getOptionLabel={(option) => option.label}
+          renderInput={(params) => <TextField {...params} variant="standard" label="Members" placeholder="Select A User" />}
+        />
+      </Grid>
+      <Grid container xs={4} md={2} lg={1}>
+        <Grid item>
+          <IconButton children={<Save />} onClick={submitMembers} />
         </Grid>
-        <Grid item xs={12}>
-          <DetailDisplay label="Members" content={team.members.map((member) => fullNamePipe(member)).join(', ')} />
+        <Grid item>
+          <IconButton children={<Cancel />} onClick={() => setIsEditingMembers(false)} />
         </Grid>
       </Grid>
+    </Grid>
+  );
+
+  const nonEditingHeadView = (
+    <Grid container flexDirection={'row'} pl={1} xs={12}>
```

Comment:
If you remove the container grid I mentioned on line 154, you shouldn't need this xs prop